volume-checker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-alpine
```

Comment:
We recommend using a multistage build for go images (example: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/run/hello-broken/Dockerfile#L38). We also recommend providing comments for each step.

Reply:
Changed to multi-stage build

Reply:
+ added comments for each step